Vagrantfile

```diff
@@ -48,6 +48,9 @@ echo "Launching bootstrap"
 exec "/srv/scality/metalk8s-$SHORT_VERSION/bootstrap.sh"
 SCRIPT
 
+NETWORK_RANGE = "192.168.42."
+BOOTSTRAP_IP_OFFSET = 10
```

Comment:
What's the need for this?

Reply:
Just to define once and for all the IP of the bootstrap node in the network range and use it as an offset for the other IP addresses. We could do without it though.